ddtrace/tracer/spancontext_test.go

```diff
@@ -143,15 +134,13 @@ func TestSpanTracePushNoFinish(t *testing.T) {
 	root.context.trace = buffer
 
 	buffer.push(root)
-	assert.Len(tracer.errorBuffer, 0)
 	assert.Len(buffer.spans, 1, "there is one span in the buffer")
 	assert.Equal(root, buffer.spans[0], "the span is the one pushed before")
 
 	select {
```

Comment:
Not a biggie, but the `select` isn't needed anymore.

Reply:
Indeed! Removed!